examples/chartmuseum/repository_chartmuseum.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: core.kubebb.k8s.com.cn/v1alpha1
+kind: Repository
+metadata:
+  name: chartmuseum
+  namespace: kubebb-system
+spec:
+  url: 10.96.116.173:8080
```

Comment:
Add protocol. 
If there is no protocol an error will be reported here.

https://github.com/kubebb/core/blob/542ef80fde088f929800bcc15eb8ff7b695a4fb0/pkg/helm/repo.go#L128